backend/src/seeds/graphql/queries.ts

```diff
@@ -135,8 +135,8 @@ export const communitiesQuery = gql`
 `
 
 export const getCommunityByUuidQuery = gql`
-  query ($communityUuid: String!) {
-    community(communityUuid: $communityUuid) {
+  query ($communityIdentifier: String!) {
+    community(communityIdentifier: $communityIdentifier) {
```

Comment:
how can you serve with this query the signature of the `CommuntityResolver.community`-method with a 2nd parameter `foreign`?
Yes the 2nd parameter is optional, but not able to set over this query. Is this intended?

Reply:
This query is in the seed folder so it is used only for tests.
It is named getCommunityByUuidQuery. I don't see why I should add foreign for a query which will from the name here only query by uuid.
But maybe you are right and I better update the tests.